main.rb

```diff
@@ -0,0 +1,142 @@
+require './person'
+require './student'
+require './teacher'
+require './classroom'
+require './book'
+require './rental'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+    @class = Classroom.new('class')
+    @list = {
+      '1' => 'List all books',
+      '2' => 'List all people',
+      '3' => 'Create a person',
+      '4' => 'Create a book',
+      '5' => 'Create a rental',
+      '6' => 'List all rentals for a given person id',
+      '7' => 'Exit'
+    }.freeze
+  end
+
+  def run
+    puts 'Please choose an option by entering a number:'
+    loop do
+      @list.each do |key, value|
+        puts "#{key} - #{value}"
+      end
+      input = gets.chomp.to_i
+      if input == 7
+        puts 'Thanks for using our app'
+        break
+      end
+      list input
+    end
+  end
+
+  def list(input)
+    case input
+    when 1
+      list_books
+    when 2
+      list_people
+    when 3
+      puts 'Do you want to add a student(1) or a teacher(2)? [input the number]:'
+      p_input = gets.chomp.to_i
+      person_input(p_input)
+    when 4
+      create_book
+    when 5
+      create_rentals
+    when 6
+      list_rentals
+    else
+      puts 'Please choose a valid number'
+    end
+  end
+
+  def person_input(person_input)
+    case person_input
+    when 1
+      add_student
+    when 2
+      add_teacher
+    else
+      puts 'Invalid number'
+    end
+  end
+
+  def add_student
+    puts 'Enter the name of student:'
+    name = gets.chomp
+    puts 'Enter the age of the student'
+    age = gets.chomp
+    puts 'parent permission? [Y/N]:'
+    parent_permission = gets.chomp.downcase
+    student = Student.new(@class, age, name, parent_permission)
+    @people << student
+    puts 'Student created successfully'
+  end
```

Comment:
- [ ] In the classroom.rb file, you are importing the wrong file which is supposed to be `./student.rb` not `./Student.rb` which makes your application raise an error.

![rb](https://user-images.githubusercontent.com/55635977/151389220-21eaabc7-b591-4a50-a136-3571265f3e0a.png)